model/task_lifecycle.go

```diff
@@ -1203,8 +1282,18 @@ func evalLinearStepback(ctx context.Context, t *task.Task, caller, status string
 
 // evalBisectStepback performs bisect stepback on the task.
 func evalBisectStepback(ctx context.Context, t *task.Task, caller, status string, deactivatePrevious bool) error {
-	// TODO: EVG-20788 implement stepback bisection.
-	return errors.Wrap(doBisectStepback(ctx, t), "performing bisect stepback")
+	// If the task is aborted or it isn't a mainline commit or existing stepback, no-op.
+	if t.Aborted || (t.ActivatedBy != evergreen.RepotrackerVersionRequester && t.ActivatedBy != evergreen.StepbackTaskActivator) {
```

Comment:
We actually already have a guarantee [here](https://gist.github.com/hadjri/ec0ec88da8a1e76c1b6b0fe4ecfc0bf2#file-task_lifecycle-go-L766) that the requester is not a patch so we can drop the check for `RepotrackerVersionRequester` (also because we do want to support `TriggerRequester` which is a different kind of requester that shows up on the mainline commit page)

So this can actually just be `if t.Aborted{ return nil }`